src/pal/src/exception/remote-unwind.cpp

```diff
@@ -62,12 +62,26 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 SET_DEFAULT_DEBUG_CHANNEL(EXCEPT);
 
-// Only used on the AMD64 build
-#if defined(_AMD64_) && defined(HAVE_UNW_GET_ACCESSORS)
+// Only used on the AMD64/ARM64 builds - ARM32 uses EXIDX which isn't currently supported
+#if (defined(_AMD64_) || defined(_ARM64_)) && defined(HAVE_UNW_GET_ACCESSORS)
 
 #include <elf.h>
 #include <link.h>
 
+#if defined(__i386) || defined(__ARM_EABI__) 
```

Comment:
I wonder why do you use `__i386`, `__x86_64__` and `__ARM_EABI__` instead of our defines (`_X86_`,  `_AMD64_` and `_ARM_`)?

Reply:
I copied the defines from createdump which uses those for some reason I don't remember (and I wrote the code). I'll fix them.